### PR TITLE
Add support for single-select-mode

### PR DESCRIPTION
**Pull request by Navas23**

### CODE REVIEW

File: src/elements/GeoTree/GeoTreeItem.vue [hunk 1]
```diff
@@ -42,11 +42,12 @@
           />
         </span>
         <input
+          v-if="!isSingleSelectMode || isSingleItem"
```
Comment (dletran) [inline src/elements/GeoTree/GeoTreeItem.vue:45]: this is to not show the input on folders when it's single select mode right?

**Review by dletran — APPROVED** [review]

File: src/elements/GeoTree/GeoTreeItem.vue (hunk 1 above)
Comment (Navas23) [reply src/elements/GeoTree/GeoTreeItem.vue:45]: Exactly

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
 # Changelog
 
+## 33.2.0
+New:
+- `GeoTree` and `GeoTreeItem`: Add support for single select mode UX. Added prop `is-single-select-mode` to use it
```
Comment (fofi) [inline CHANGELOG.md:5]: LEts update this comment and prop comment to make more clear that the new prop is to use radio vs checkbox input

File: src/elements/GeoTree/GeoTreeItem.vue [hunk 3]
```diff
@@ -42,11 +42,12 @@
...
           :id="category[keyForId]"
           :checked="isChecked"
           :indeterminate.prop="isIndeterminate"
```
Comment (fofi) [inline src/elements/GeoTree/GeoTreeItem.vue:48]: Not sure if a good idea, but we could stop calculating some computed properties that do not make sense on single mode

Comment (Navas23) [reply src/elements/GeoTree/GeoTreeItem.vue:48]: IMO is not worth it for this case

**Review by fofi — APPROVED** [review]

**Pull request merged**